### DISCUSSION TITLE
Is there a way to include the relation manager table in a modal view page?

### DISCUSSION THREAD

**Discussion by nadeaumarc** [filamentphp/filament#11990] [Help]
### Package

Panel builder

### Package Version

v3.2.5

### How can we help you?

Hey!

I was wondering if there is a way to display the relation manager's table inside of a view modal? (Without the view page)

```
->actions([
    ViewAction::make()
        ->modalWidth(MaxWidth::FiveExtraLarge),
])
```

**Comment by torgodly** [comment]
hey ,, did you find a way?

**Comment by ahmed-fawzy99** [comment]
I was looking for this as well. Did you find a way to do it?

**Comment by jeangomes** [comment]
Is there any news on this question?

**Comment by opheus2** [comment]
https://filamentphp.com/plugins/guava-modal-relation-managers

Isn't this package solving that?

**Reply by KamranBiglari** [reply]
No
